glib-macros/tests/properties.rs

```diff
@@ -145,6 +146,8 @@ mod foo {
             cell: Cell<u8>,
             #[property(get = Self::overridden, override_class = Base)]
             overridden: PhantomData<u32>,
+            #[property(name = "thread-guard", get, set)]
```

Comment:
No need to manually set the name. The macro generates it for you. The underscore _ becomes a dash -.